cocos/core/gfx/webgl/webgl-queue.ts

```diff
@@ -23,7 +24,7 @@ export class WebGLGFXQueue extends GFXQueue {
         this._status = GFXStatus.UNREADY;
     }
 
-    public submit (cmdBuffs: GFXCommandBuffer[], fence?) {
+    public submit (cmdBuffs: GFXCommandBuffer[], fence?: GFXFence) {
```

Comment:
Can we remove fence here? gfx internally handle the fence issue.

Reply:
No this is where the multi-threaded context may take advantage of, by passing fence in one thread and wait for it in another.